compiler/src/iree/compiler/Dialect/Flow/Transforms/SplitReduction.cpp

```diff
@@ -27,6 +28,10 @@ static llvm::cl::opt<int64_t> splitReductionRatio(
     "iree-flow-split-matmul-reduction", llvm::cl::desc("split ratio"),
     llvm::cl::init(1));
 
+static llvm::cl::opt<int64_t> topkSplitReductionRatio(
+    "iree-flow-topk-split--reduction", llvm::cl::desc("split ratio"),
```

Comment:
Why is there a `--`? SHould be `iree-flow-topk-split-reduction`

Reply:
necessary for flags to work. Without it, its read as a positional argument.

Reply:
I don't think you need `--` within the name

Reply:
oh, my bad, let me fix that